src/utils/risk-framework.ts

```diff
@@ -1,7 +1,7 @@
 import riskFrameworkJson from './config/risk-framework.json';
 
 export const initRiskFrameworkScores = (network: string) => {
```

Comment:
LGTM, maybe for a future PR can the network type here be either  a string or string[], that way we can support an array, i think its cleaner than separator syntax?